src/settings.ts

```diff
@@ -748,3 +748,10 @@ export const PROFILE_MODAL_EDIT_MODE_TURNING_OFF_DELAY = 130;
 export const EVENT_STARTING_SOON_TIMEFRAME = 120; // in minutes
 
 export const EVENTS_PREVIEW_LIST_LENGTH = 5;
+
+// Set these to have images uploaded to Firebase Storage served off of Imgix
+// @debt load this from an env variable. This is good enough for Burning Man but we want to have env-specific conf
+export const FIREBASE_STORAGE_IMAGES_ORIGIN =
+  "https://firebasestorage.googleapis.com/v0/b/sparkle-burn.appspot.com/o/";
+export const FIREBASE_STORAGE_IMAGES_IMGIX_URL =
+  "https://sparkle-burn-users.imgix.net/";
```

Comment:
We are merging this into `staging`, aren't we? It means that it'll be propagated to every env

Reply:
This will only impact assets stroed under ` "https://firebasestorage.googleapis.com/v0/b/sparkle-burn.appspot.com/o/";`. So no, it won't have any effect outside the `sparkle-burn` env until we take care of the `@debt` item